frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/SelectedNodeInputOutputTab.tsx

```diff
@@ -19,6 +23,17 @@ const SelectedNodeInputOutputTab: React.FC<SelectedNodeInputOutputTabProps> = ({
   task,
   execution,
 }) => {
+  const { namespace } = usePipelinesAPI();
+  const isExperimentsAvailable = useIsAreaAvailable(SupportedArea.PIPELINE_EXPERIMENTS).status;
+
+  const getExecutionDisplayNameFromObject = React.useCallback(
```

Comment:
this should not use callback. you are using it as a value so just use useMemo

Reply:
Yes, i was using it as a function before, now i've changed it to a variable